src/test/java/com/razorpay/CardClientTest.java

```diff
@@ -0,0 +1,60 @@
+package com.razorpay;
+
+import org.junit.Test;
+import org.mockito.InjectMocks;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class CardClientTest extends BaseTest{
+
+    @InjectMocks
+    protected CardClient cardClientClient = new CardClient(TEST_SECRET_KEY);
+
+    private static final String CARD_ID = "card_DZon6fd8J3IcA2";
+
+    private static final String PAYMENT_ID = "pay_IDRP0tbirMSsbn";
+
+    /** Fetch card details
+     * @throws RazorpayException
+     */
+    @Test
+    public void fetch() throws RazorpayException {
+
+        String mockedResponseJson = "{\n  \"id\": "+CARD_ID+",\n  \"entity\": \"card\",\n  \"international\": false,\n  \"last4\": 1111,\n  \"name\": \"sample name\",\n  \"network\": \"Visa\",\n  \"type\": \"debit\"\n}";
+        try {
+            mockResponseFromExternalClient(mockedResponseJson);
+            mockResponseHTTPCodeFromExternalClient(200);
+            Card fetch = cardClientClient.fetch(CARD_ID);
+            assertNotNull(fetch);
+            assertEquals(CARD_ID,fetch.get("id"));
+            assertTrue(fetch.has("international"));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
e.printStackTrace should be removed

Reply:
should i add assertTrue(False) instead of e.printStackTrace ?